main.js

```diff
@@ -123,6 +128,8 @@ class Tado extends utils.Adapter {
 						const power = await this.getStateAsync(home_id + '.Rooms.' + zone_id + '.setting.power');
 						const durationInSeconds = await this.getStateAsync(home_id + '.Rooms.' + zone_id + '.overlay.termination.durationInSeconds');
 						const nextTimeBlockStart = await this.getStateAsync(home_id + '.Rooms.' + zone_id + '.nextTimeBlock.start');
+						const openWindowDetectionEnabled = await this.getStateAsync(home_id + '.Rooms.' + zone_id + '.openWindowDetection.enabled');
```

Comment:
is there a reason not to put this in a separate "else if" block as  for 'childLockEnabled'?
In the last block all states are handled which trigger setZoneOverlay().
I suggest to add additional 'else if' blocks above...

Reply:
Okay, I wasn't sure about that. I added two new if-blocks for `activateOpenWindow` and `openWindowDetection`-settings